proposals/lip-0057.md

```diff
@@ -160,10 +173,10 @@ The store keys and values of the DPoS store are set as follows:
 
 ##### Store Prefix, Store Key, and Store Value
 
-* The store prefix is set to `STORE_PREFIX_VOTER`.
+* The store prefix is set to `SUBSTORE_PREFIX_VOTER`.
 * Each store key is a `ADDRESS_LENGTH`-byte `address`, representing a user address.
 * Each store value is the serialization of an object following `voterStoreSchema`.
-* Notation: For the rest of this proposal let `voterStore(address)` be the object stored in the voter substore with store key `address`.
+* Notation: For the rest of this proposal let `voterStore(address)` be the object stored in the voter substore with store key `address`, deserialized as a dictionary.
```

Comment:
As also raised on Slack, I am a bit confused with this changed store notation.

Reply:
Replied to this discussion, will take into account on the next push.

Reply:
Looks good now, I resolve this comment.